lms/static/sass/course/modules/_calculator.scss

```diff
@@ -125,19 +144,31 @@ div.calc-main {
           right: 0;
           top: 0;
 
+          #hint-instructions {
+            color: $white;
+          }
+
           #calculator_hint {
-            background: url("../images/info-icon.png") center center no-repeat;
-            height: 35px;
             @include hide-text;
-            width: 35px;
             display: block;
+            height: 35px;
+            width: 35px;
             border: none;
+            background: url("../images/info-icon.png") center center no-repeat;
+            color: $white;
 
             &:focus {
               outline: 5px auto #5b9dd9;
               box-shadow: none;
               border: none;
             }
+
+            span.sr {
```

Comment:
Hmm, any chance we can give this button another class (like ``calculator_hint-label``) so we can keep the rules scoped, but tied to a class rather than an element?